### DISCUSSION TITLE
What conditions are used to determine the cycle of insufficient resources but continuous attempts to allocate them?

### DISCUSSION THREAD

**Discussion by shijili** [manoelcampos/cloudsim-plus#488] [Q&A] [answered]
I am using cloudsimplus as a spring boot back-end application. How can I detect the following situation in the simulation and terminate it in time so that I can send prompt information to the front-end?
![image](https://github.com/cloudsimplus/cloudsimplus/assets/155445121/6a713c29-86fd-4ccb-9483-b7846e1cb7b9)

**Comment by manoelcampos** [comment, marked as answer]
Try broker.getVmCreation().setMaxRetries(0);

**Reply by shijili** [reply]
Thank you so much! I looked at the VmCreation class in detail, and your answer can indeed terminate this endless loop. I tried to use "broker0.getVmCreation().setMaxRetries(3)" and an infinite loop appeared again, which I couldn't understand.It seems that this parameter can only be set to 0. Anything greater than 0 will not achieve the expected effect. Sometimes the program needs to try to create a virtual machine again.Setting all to 0 may not achieve the expected results in some scenarios.
